example.R

```diff
@@ -36,10 +41,14 @@ output.high <- corr.bestlag(data = num.data, timepoints = c(0.0001, 0.5, 3, 6, 1
 
 ## analyzing time series data of 100 genes using LPWC with low penalty,
 ## the low penalty allows more lags
-# *** TODO: can we demonstrate how it allows more lags by showing the number of nonzero lags? ***
 output.low <- corr.bestlag(data = num.data, timepoints = c(0.0001, 0.5, 3, 6, 12), 
                            penalty = "low")
 
+## demonstrating more lags with low penalty
+## comparing the lags output for low and high 
+table(output.high$lags)
```

Comment:
Can you add row names to the table or more comments to explain the output?  Seeing
```
-1  0  1 
 2 95  3 
```
won't make sense to a new users.

We also get only a few more non-zero lags with the low penalty.  It makes it seem like the low penalty isn't very different from the high penalty.

Reply:
The `table(output.high$lags)` style is better because it summarizes the lag counts, but it still is confusing without row labels.  I'll edit the comments.